### ISSUE TITLE
How to set hyper-parameters of GCNN-13(Dauphin et al. (2017): Language Modeling with Gated Convolutional Networks)? 

### ISSUE THREAD

**Issue by cswangjiawei** [pytorch/fairseq#734]
How to set hyper-parameters of GCNN-13(Dauphin et al. (2017): Language Modeling with Gated Convolutional Networks)? Such as dropout ratio, optimizer, learn rate, max-epoch, initialization of layers of the model, gradient clipping value, weight-decay.

**Comment by huihuifan** [comment]
When I initially ran the experiments in LuaTorch, I used the following parameters for GBW: 0 dropout, nag optimizer, learning rate 1, kaiming initialization, clipping 0.1, no weight decay
